src/Core/StackUri.php

```diff
@@ -146,17 +153,30 @@ public function addOverridingOptions($options)
                     continue;
                 }
                 $parsedOptions = self::parseOptions(\array_slice($stringOperationWithOptions, 1));
-                if ('options' === $stringOperationName) {
+                if ('options' === $stringOperationName || 'o' === $stringOperationName) {
                     $this->setStackOptions(array_merge($this->getStackOptions(), $parsedOptions));
+                } elseif ('variables' === $stringOperationName || 'v' === $stringOperationName) {
+                    $this->setStackVariables(array_merge($this->getStackVariables(), $parsedOptions));
                 } else {
-                    // only add as stack operation everything before the first /
+                    // move options with [] as start and end to expressions
+                    $expressions = [];
+                    foreach ($parsedOptions as $_k => $_v) {
+                        if ('[' === substr($_v, 0, 1) && ']' === substr($_v, -1, 1)) {
+                            $expressions[$_k] = substr($_v, 1, -1);
+                            unset($parsedOptions[$_k]);
```

Comment:
I'd continue here, and have another if case below, but that's a matter of taste.

Reply:
also changed